one_step_setup

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+set -e
+
+function fail {
+    # Print a newline to stderr and exit.
+    echo > /dev/stderr
+    exit 1
+}
+
+
+# `$PANTS_VERSION` must be set
+if [ -z "$PANTS_VERSION" ]
+then
+    echo \`\$PANTS_VERSION\` must be set. Finish installing Pants by re-running this script \
```

Comment:
This gets you close:

```
cat /tmp/json | grep -o '"version":"[^"]*"'
```

Or writing this in Python...

Reply:
@benjyw Nice. I can work with that.

Reply:
@benjyw This seems to work just fine:

```
PANTS_VERSION=`curl https://pypi.org/pypi/pantsbuild.pants/json  | grep -o '"version":"[^"]*"' | grep -o "[0-9]*\\.[0-9]*\\.[0-9]*"`
```

Reply:
As the person who took away this affordance in https://github.com/pantsbuild/setup/pull/86...I am okay with it because we are going to be pinning the version in `pants.toml`.

We probably want on dev versions to have people set the `PANTS_VERSION` explicitly though? So our docs still need to mention this env var?

Reply:
That seems like a good idea. I'll add support for setting the variable still.

Reply:
@Eric-Arellano @benjyw I've added this feature.